storage/database/shardedPersister_test.go

```diff
@@ -342,6 +344,7 @@ func getKey(
 	id string,
 ) {
 	go func(key []byte) {
```

Comment:
the create persister should not be done on goroutine as you might not finish the creation/opening and already reach the get on line 359

Reply:
done